app/javascript/components/calendar/utils.js

```diff
@@ -197,8 +197,30 @@ const setupRealtimeRefresh = (fullCalendarInstance, agentIds) => {
     }
   };
 
+  const disconnectCallback = () => {
+    clearTimeout(window.disconnectWarningTimeoutId);
```

Comment:
je ne sais pas ce qu’il se passe quand on fait `clearTimeout(null)`, éventuellement :

```suggestion
    if (window.disconnectWarningTimeoutId) { clearTimeout(window.disconnectWarningTimeoutId); }
```

Reply:
À la base j'avais mis le if, et puis j'ai lancé `clearTimeout(null)` dans mon navigateur et ça fait rien du tout donc j'(ai préféré simplifier. Je suis d'accord que ça explicite moins bien l'intention du code, mais je crois que je préfère quand-même la concision ici vu que c'est pas trop complexe.